### DISCUSSION TITLE
Limit bucket size or subfolders in a bucket

### DISCUSSION THREAD

**Discussion by manigarg31** [minio/minio#14238] [Q&A] [answered]
is there a way to expire objects on the basis of bucket size or sub folder size ?

**Comment by harshavardhana** [comment, marked as answer]
@manigarg31 use ILM expiry.

**Reply by manigarg31** [reply]
ILM expiry provides on the basis of days only it doesn't provide it on the basis of size

**Reply by harshavardhana** [reply]
Use expiry for now @manigarg31 - we may add based on size in future.

**Reply by manigarg31** [reply]
Cool, thanks.
